_dev/src/assets/json/translations/en/ui.json

```
@@ -239,14 +239,18 @@
       "shippingSettingsErrors": "This country is not compatible with your currency",
       "syncSchedule" : "Sync schedule"
     },
+    "inputTargetCountriesTooltip": "Note that you can't select a country with a different currency than your shop’s",
```

```suggestion
    "inputTargetCountriesTooltip": "Note that you can't select a country with a different currency than your shop's",
```

Let's use the common single quote